src/classes/OPP_OpportunityNaming_BATCH.cls

```diff
@@ -35,56 +35,29 @@
 * @description Batch class creates names all Opportunities per the naming spec. Batch job chunks Opportunities
 * in groups of 10,000,000 records ordered by Id and chains itself to process additional records to avoid query limits
  */
-public class OPP_OpportunityNaming_BATCH implements Database.Batchable<sObject>, Schedulable, Database.Stateful {
-    /** @description The query for the batch process to run on.*/
-    private String query;
-
-    @TestVisible
-    private static Integer defaultQueryLimit = 10000000;
+public class OPP_OpportunityNaming_BATCH extends UTIL_AbstractCommon_BATCH implements Schedulable {
 
-    @TestVisible
-    private Id lastOppIdProcessed;
-
-    /** @description The batch process constructor; creates opportunity query for all opportunities.*/
+    /** @description Default No argument constructor */
     public OPP_OpportunityNaming_BATCH() {
-        query = 'SELECT Id, Name FROM Opportunity ORDER BY Id LIMIT ' + defaultQueryLimit;
-    }
-
-    /** @description Constructor that accepts Id offset*/
-    public OPP_OpportunityNaming_BATCH(Id idToOffset) {
-        query = buildOffsetQuery(idToOffset, defaultQueryLimit);
+        super();
     }
 
-    /** @description Builds query string with given Id to offset and query limit*/
-    private String buildOffsetQuery(Id idToOffset, Integer queryLimit) {
-        return String.format(
-            'SELECT Id, Name FROM Opportunity WHERE Id > \'\'{0}\'\' ORDER BY Id LIMIT {1}',
-            new List<String>{ idToOffset, String.valueOf(queryLimit) }
-        );
-    }
-
-    /** @description Batch process start method.*/
-    public Database.QueryLocator start(Database.BatchableContext BC) {
-        return Database.getQueryLocator(query);
+    /** @description Constructor to allow specifying batch size.*/
+    public OPP_OpportunityNaming_BATCH(Integer batchSize) {
+        super(batchSize);
     }
 
     /** @description Schedulable execute method.*/
     public void execute(SchedulableContext context) {
-        Database.executeBatch(new OPP_OpportunityNaming_BATCH(), 200);
+        Database.executeBatch(new OPP_OpportunityNaming_BATCH(200), 200);
```

Comment:
Minor request. Can you make the 200 batch size a constant at the top of this class?
`private static final Integer BATCH_SIZE = 200;`